src/vm/typehandle.inl

```diff
@@ -283,6 +283,53 @@ inline void TypeHandle::ForEachComponentMethodTable(T &callback) const
     }
 }
 
+#ifndef CROSSGEN_COMPILE
+FORCEINLINE OBJECTREF TypeHandle::GetManagedClassObjectFast() const
+{
+    CONTRACTL{
+        NOTHROW;
+        GC_NOTRIGGER;
+        MODE_ANY;
 
-#endif  // _TYPEHANDLE_INL_
+        FORBID_FAULT;
+    }
+    CONTRACTL_END;
+
+    OBJECTREF o = NULL;
 
+    if (!IsTypeDesc()) {
+        o = AsMethodTable()->GetManagedClassObjectIfExists();
+    }
+    else
+    {
```

Comment:
Nit: seems to have an inconsistent style about where the `{` goes, with it being on the same line as the `if`